srcs/layouts/guj_phonetic_in.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="ગુજરાતી ફોનેટિક - Gujarati Phonetic" script="gujarati">
+  <row>
+    <key key0="1" key1="`" key2="૧" key3="!" key4="esc"/>
+    <key key0="2" key1="~" key2="૨" key3="\@" key4="૱"/>
```

Comment:
The layout contains a lot of ASCII symbols like `~`, `@`, `^`, etc.. Are these really used ?

Reply:
mostly not much useful frequently but kept it as default layout has it and most keyboards contains these.

Reply:
Makes sense.